### PR TITLE
Fix/27997

### PR DESCRIPTION
**Pull request by abhinav-from-contentstack**
CS-27997

### CODE REVIEW

**Review by netrajpatel — APPROVED** [review]

File: packages/contentstack-export-to-csv/src/commands/cm/export-to-csv.js [hunk 1]
```diff
@@ -192,26 +207,29 @@ class ExportToCsvCommand extends Command {
             util.write(this, listOfUsers, fileName, 'organization details');
           } catch (error) {
             if (error.message) {
-              this.log(`Error: ${error.message}`);
+              this.log(`${util.formatError(error)}`);
```
Comment (antonyagustine) [inline packages/contentstack-export-to-csv/src/commands/cm/export-to-csv.js:210]: @abhinav-from-contentstack this string concatenation is not necessary. We can remove it here. this.log(util.formatError(error));

File: packages/contentstack-export-to-csv/src/commands/cm/export-to-csv.js [hunk 2]
```diff
@@ -192,26 +207,29 @@ class ExportToCsvCommand extends Command {
...
-
-            this.error('failed export content to csv');
           }
           break;
         }
       }
     } catch (error) {
       if (error.message) {
-        this.log(`Error: ${util.formatError(error)}`);
+        this.log(`${util.formatError(error)}`);
```
Comment (antonyagustine) [inline packages/contentstack-export-to-csv/src/commands/cm/export-to-csv.js:218]: @abhinav-from-contentstack this string concatenation is not necessary. We can remove it here. this.log(util.formatError(error));

**Review by antonyagustine — CHANGES_REQUESTED** [review]

File: packages/contentstack-export-to-csv/src/commands/cm/export-to-csv.js (hunk 1 above)
Comment (abhinav-from-contentstack) [reply packages/contentstack-export-to-csv/src/commands/cm/export-to-csv.js:210]: Hi, I've removed the string concatenation.

File: packages/contentstack-export-to-csv/src/commands/cm/export-to-csv.js (hunk 2 above)
Comment (abhinav-from-contentstack) [reply packages/contentstack-export-to-csv/src/commands/cm/export-to-csv.js:218]: Hi, I've made the requested change.

**Review by antonyagustine — APPROVED** [review]

**Pull request merged**